neurons/validators/src/miner_jobs/machine_scrape.py

```diff
@@ -516,6 +517,53 @@ def get_network_speed():
         data["network_speed_error"] = repr(exc)
     return data
 
+def get_container_id():
+    """Retrieve the Docker container ID if running inside a container."""
+    try:
+        with open('/proc/self/cgroup', 'r') as f:
+            for line in f:
+                if 'docker' in line:
+                    return line.strip().split('/')[-1]
+    except FileNotFoundError:
+        return None
+    
+def get_container_digest(container_id):
```

Comment:
@LegendWorrier I don't think we need this function

Reply:
okay, will remove it.